inductiva/tasks/task.py

```diff
@@ -401,16 +414,56 @@ def download_outputs(
             warnings.warn("Path already exists, files may be overwritten.")
         output_dir.mkdir(parents=True, exist_ok=True)
 
+        download_message = "Downloading simulation outputs to %s"
+
+        if filenames:
```

Comment:
Thanks Bruno, this function is becoming a bit too complex with lots of conditional logic, do you think you can simplify it a bit?

Reply:
Sure, thanks for the suggestions.